monitoring/ceph-mixin/tests_alerts/test_alerts.yml

```diff
@@ -1754,15 +1754,15 @@ tests:
    alert_rule_test:
     # PG_AVAILABILITY and OSD_DOWN not firing .. no alert
     - eval_time: 1m
-      alertname: CephPGUnavilableBlockingIO
+      alertname: CephPGUnavailableBlockingIO
```

Comment:
The alert names are more likely to be problematic, so I think it's likely the project will want to drop this

Reply:
I'm happy with this being fixed. IMHO CamelCase alert names (formerly we used snake_case) reduced readability and made it easier to make mistakes.